packages/block-library/src/blocks/supernova/edit.js

```diff
@@ -103,10 +103,9 @@ const SupernovaPreview = props => {
 
   return (
     <div { ...blockProps }>
-      {/*<div className={ `wp-block__inner-container` }>*/}
         {
           headerPosition === 0 && ( showCollectionTitle || showCollectionSubtitle ) &&
-          <div className="wp-block" data-align={ align }>
+          <div className="wp-block supernova-header__inner-container" data-align={ align }>
```

Comment:
The pattern we should stick to is `wp-block` elements are wrapped in `__inner-container` elements for the spacing CSS rules to work consistently.

Reply:
My mistake, I forgot to remove wp-block from `supernova-header__inner-container`. We should be good now.